databricks/sdk/core.py

```diff
@@ -931,8 +919,7 @@ def __init__(self, cfg: Config = None):
         # Prevents platform from flooding. By default, requests library doesn't block.
         pool_block = True
 
-        http_adapter = HTTPAdapter(max_retries=retry_strategy,
-                                   pool_connections=pool_connections,
+        http_adapter = HTTPAdapter(pool_connections=pool_connections,
```

Comment:
The `max_retries` parameter is for failed DNS lookups, or connection attempts (see https://requests.readthedocs.io/en/latest/api/#requests.adapters.HTTPAdapter), not retries once a connection has been made. The retries you're dealing with below are relevant for once a connection has been made.